src/main/java/frc/trigon/robot/subsystems/roller/simulationroller/SimulationRollerConstants.java

```diff
@@ -0,0 +1,39 @@
+package frc.trigon.robot.subsystems.roller.simulationroller;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import frc.trigon.robot.motorsimulation.MotorSimulationConfiguration;
+import frc.trigon.robot.motorsimulation.SimpleMotorSimulation;
+
+public class SimulationRollerConstants {
+    public static double VOLTAGE_COMPENSATION_SATURATION = 12;
+
+    private static final double CONVERSION_FACTOR = 1;
+    private static final int NUMBER_OF_MOTORS = 1;
+    private static final DCMotor MOTOR_GEARBOX = DCMotor.getKrakenX60Foc(NUMBER_OF_MOTORS);
+    private static final double GEAR_RATIO = 1;
+    private static final double MOMENT_OF_INERTIA = 0.003;
+    static final SimpleMotorSimulation MOTOR_SIMULATION = new SimpleMotorSimulation(
+            MOTOR_GEARBOX,
+            GEAR_RATIO,
+            MOMENT_OF_INERTIA
+    );
+
+    private static final double
+            KS = 0,
+            KV = 0,
+            KA = 0;
+
+    static {
+        final MotorSimulationConfiguration config = new MotorSimulationConfiguration();
+
+        config.voltageCompensationSaturation = VOLTAGE_COMPENSATION_SATURATION;
+
+        config.conversionFactor = CONVERSION_FACTOR;
```

Comment:
No need for this newline. Also, there's a typo. It should be conversionsFactor and not conversionFactor